.github/workflows/pr-commands.yaml

```diff
@@ -0,0 +1,75 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
```

Comment:
This is a note for myself, but this action rebuilds the documentation and styles when submitting a pull request